src/pages/aboutme.tsx

```diff
@@ -0,0 +1,45 @@
+
+
+export default function AboutMe() {
+    return (
+        <div className="aboutMePage">

```

Comment:
Take a look at this demo https://codesandbox.io/s/sleepy-galois-uwqpmo?file=/src/styles.css